src/plays/countries-statics/SearchAndFilter.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { BsSearch } from "react-icons/bs";
+
+export default function SearchAndFilter() {
```

Comment:
I don't think this is wired up. can you remove and add back when you are going to introduce search?